raiden-ts/src/utils/types.ts

```diff
@@ -221,34 +221,50 @@ export const Address: AddressC = new t.RefinementType<HexStringC<20>, Address, s
 );
 
 /**
- * Helper function to create codecs to validate [timestamp, value] tuples
+ * Helper type to extend a given type T to contain a timestamp ts member
+ */
+export type Timed<T> = T & { readonly ts: number };
```

Comment:
Do you think `ts` is sufficient enough? :thinking: 
In future comment you always state `a ts timestamp property`. So does `ts` mean something else than **t**ime**s**tamp?

Reply:
`ts` is the abbreviation for timestamp here, and the name of the property. `timestamp` on this comment is used as a _[semantic] type_ for the property: i.e. a number which contains unix epoch value in milliseconds. But yes, they're the same thing, just wanted to be more descriptive on the type docstring.

Reply:
Hmm. I would loved to have `timestamp` as the property name. :sweat:

Reply:
`ts` is very commonly used as abbreviation of `timestamp`, including on the linux kernel. don't suffer for it xD

Reply:
You definitely got me with the argument that the Linux kernel does the same. :grimacing: